packages/utils/src/mock-webhook.ts

```diff
@@ -1,11 +1,32 @@
 import express, { Application, Request, Response } from "express";
+import crypto from "crypto";
 
 const app: Application = express();
+
 app.use(express.json({ limit: "2mb" }));
 app.use(express.urlencoded({ extended: false, limit: "2mb" }));
 
+const secretKey = process.argv[2]; // Get the secret key from command line arguments
```

Comment:
for testing

Reply:
see other utils, we use env vars for that (`getEnvVar()` | `getEnvVarOrFail()`) - which allows to store values on .env and not having to provide those on the command line